src/org/freedesktop/gstreamer/query/AllocationQuery.java

```diff
@@ -85,7 +85,7 @@ public Caps getCaps() {
     }
 
     // @TODO how best not to expose GType?
-    void addAllocationMeta(GType api, Structure params) {
+    public void addAllocationMeta(GType api, Structure params) {
```

Comment:
We cannot expose GType in the API here - need an alternative, probably based around Meta.API. But need to think how to handle fact this is in a separate package.

Reply:
I use now Meta.API and add a new Meta.API::getAPITypeName() method before finding another solution